tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -26,6 +26,8 @@ all: serial-targets \
 	kernel-tarball \
 	kernel-dragonball-experimental-tarball \
 	kernel-tdx-experimental-tarball \
+	kernel-gpu-snp-tarball \
+	kernel-gpu-tdx-tarball \
```

Comment:
Same comment I made about the "experimental" bits, and also about having this for the "vanilla" kernel.